src/pages/api/update-manual-review.js

```diff
@@ -49,38 +49,41 @@ const clearRemark = async (id) => {
 
 //function for manual review
 const updateManualRepo = async (req, res) => {
-  const repoId = req.query.id;
+  let repoIds = req.query.ids.split(",");
+  repoIds = repoIds.map(Number);
```

Comment:
Why do we need this? Can this not be avoided?

Reply:
repo ids comes as a string array from request, we need to convert into in integer array